pallets/creator-staking/src/types/mod.rs

```diff
@@ -68,19 +55,23 @@ pub struct CreatorStakeInfo<Balance: HasCompact + MaxEncodedLen> {
     pub total: Balance,
```

Comment:
In conversation you said that renamed it to "StakesInfo", but you renamed "StakesInfo" to "StakesInfoOf" in the PR. I am confused :)